.eslintrc.js

```diff
@@ -367,6 +368,7 @@ module.exports = {
         '@typescript-eslint/no-empty-function': 'off',
         'local-rules/nbsp-in-text': 'off',
         'local-rules/no-empty-arrow-function': 'error',
+        'local-rules/no-null-literals-template': 'off',
```

Comment:
pourquoi a-t-on besoin de l'autoriser dans les tests ?

Reply:
Tu as des `${null}` dans des `it.each`